### PR TITLE
added fetch call, changed route name, corrected card information

### PR DESCRIPTION
**Pull request by DFWilliams**

### CODE REVIEW

File: app/javascript/components/App.js [hunk 1]
```diff
@@ -26,8 +26,9 @@ class App extends Component {
   }
 
   componentDidMount(){
+    Promise.all(this.readNomaddaDestinations(),
```
Comment (elysemontano) [inline app/javascript/components/App.js:29]: It looks like readNomaddaDestinations is called twice in here.  Do you need both?

File: app/javascript/components/App.js [hunk 2]
```diff
@@ -103,6 +113,9 @@ class App extends Component {
         <Router>
           <Header {...this.props}/>
           <Switch>
+            <Route path="/countrieslist" render = {(props) => <Index {...props} countries={this.state.countries} />} />  
+            <Route exact path="/" component={Home} />
+            <Route component={NotFound}/>
```
Comment (elysemontano) [inline app/javascript/components/App.js:118]: I don't see a path for NotFound in this route.

File: app/javascript/components/App.js [hunk 3]
```diff
@@ -103,6 +113,9 @@ class App extends Component {
         <Router>
           <Header {...this.props}/>
           <Switch>
+            <Route path="/countrieslist" render = {(props) => <Index {...props} countries={this.state.countries} />} />  
+            <Route exact path="/" component={Home} />
```
Comment (elysemontano) [inline app/javascript/components/App.js:117]: I am seeing this route twice both here and on line 119.

**Review by elysemontano — CHANGES_REQUESTED** [review]
A few comments to look at.

File: app/javascript/components/App.js (hunk 1 above)
Comment (DFWilliams) [reply app/javascript/components/App.js:29]: I had to call it with the readNomaddaCountries in the Promise.all() in order  to get the country data.

**Pull request closed without merging**